p8s.go

```diff
@@ -107,6 +120,12 @@ func InitMetrics(additionalLabels ...string) *prometheus.Registry {
 		requestLabels = append(requestLabels, geoHash)
 	}
 
+	if idx := slices.Index(sanitizedP8sLabels, hostnameLabel); idx > -1 {
```

Comment:
nit:
>A little copying is better than a little dependency.

https://go-proverbs.github.io/

```go
func Index[E comparable](s []E, v E) int {
	for i, vs := range s {
		if v == vs {
			return i
		}
	}
	return -1
}
```
[src](https://cs.opensource.google/go/x/exp/+/master:slices/slices.go;l=104-113)

Reply:
So then we have various permutations of that function that subtly differ across our 50 different apps. No, I pretty fundamentally disagree with that proverb.  left-pad notwithstanding I've never yet been convinced the costs of repeated copying is cheaper than the costs of dependency.